apps/api/src/app/topics/topics.controller.ts

```diff
@@ -204,13 +214,52 @@ export class TopicsController {
       FilterTopicsCommand.create({
         environmentId: user.environmentId,
         key: query?.key,
+        subscriberId: query?.subscriberId,
         organizationId: user.organizationId,
         page: query?.page,
         pageSize: query?.pageSize,
       })
     );
   }
 
+  @Get('/subscribers/:subscriberId')
```

Comment:
Is this new endpoint should be part of this PR? i did not find any mentions of this in the tagged issue.

Reply:
@djabarovgeorge  The new endpoint was proposed by @p-fernandez on the previously raised PR, where we even had to revert another implementation for this 
see comment for reference https://github.com/novuhq/novu/pull/3551#discussion_r1227969854

The idea was proposed so we could have a simplified implementation rather than compounding two distinct implementations into one
cc: @jainpawan21